packages/fx-core/src/plugins/resource/identity/index.ts

```diff
@@ -253,9 +239,44 @@ export class IdentityPlugin implements Plugin {
   }
 
   private syncArmOutput(ctx: PluginContext) {
-    ctx.config.set(Constants.identityName, getArmOutput(ctx, IdentityArmOutput.identityName));
+    const resourceId = getArmOutput(ctx, IdentityArmOutput.identityName);
+    const resourceGroup = getResourceGroupNameFromResourceId(resourceId!);
+
+    ctx.config.set(Constants.identityName, resourceId);
     ctx.config.set(Constants.identityId, getArmOutput(ctx, IdentityArmOutput.identityId));
     ctx.config.set(Constants.identity, getArmOutput(ctx, IdentityArmOutput.identity));
+    ctx.config.set(Constants.resourceGroupName, resourceGroup);
```

Comment:
so we set both resource id and resourcegroup?

Reply:
If developers use customized resource group, it could be clear to show it in output.
As the resource id contains the resource group,  we can remove the resource group as well.